src/ParcelRegistry.Projections.Extract/ParcelExtract/ParcelExtractProjections.cs

```diff
@@ -15,21 +15,21 @@ namespace ParcelRegistry.Projections.Extract.ParcelExtract
 
     [ConnectedProjectionName("Extract percelen")]
     [ConnectedProjectionDescription("Projectie die de percelen data voor het percelen extract voorziet.")]
-    public class ParcelExtractV2Projections : ConnectedProjection<ExtractContext>
+    public class ParcelExtractProjections : ConnectedProjection<ExtractContext>
```

Comment:
@ArneD What impact does renaming the project have on the current ProjectionState?